packages/expo-updates/ios/EXUpdates/AppLoader/FileDownloader.swift

```diff
@@ -867,14 +777,14 @@ public final class FileDownloader {
     extensions: [String: Any],
     certificateChainFromManifestResponse: String?,
     database: UpdatesDatabase,
-    isVerified: Bool,
     successBlock: ParseManifestSuccessBlock,
     errorBlock: ParseManifestErrorBlock
   ) {
     var mutableManifest = manifest
+
+    // There are a few cases in Expo Go where we still want to use the unsigned manifest anyway, so don't mark it as unverified.
```

Comment:
For future investigation: Searching the code, it looks like `expectsSignedManifest` and `EXUpdatesConfig.EXUpdatesConfigExpectsSignedManifestKey` are set to YES in Expo Go. In fact, it seems to be non-Expo Go clients -- both development and production builds -- that don't require signing unless code signing is required.

Reply:
https://linear.app/expo/issue/ENG-10975/follow-up-on-expectssignedmanifest